src/jit/codegenxarch.cpp

```diff
@@ -6009,6 +6031,11 @@ void CodeGen::genGetStructTypeSizeOffset(const SYSTEMV_AMD64_CORINFO_STRUCT_REG_
             *size0 = EA_GCREF;
             *type0 = TYP_REF;
             break;
+        case SystemVClassificationTypeIntegerByReference:
```

Comment:
Added utility method in compiler.cpp that does the work and using that in both places the Jit type and size are being calculated.